source/modules/MEI.cmn.xml

```diff
@@ -291,9 +291,11 @@
         </datatype>
       </attDef>
       <attDef ident="slope" usage="opt">
-        <desc xml:lang="en">Records the slope of the beam.</desc>
+        <desc xml:lang="en">Records the slope of a beam; e.g., in MEI virtual units. Positive values indicate 
```

Comment:
Is it really "e.g., in MEI virtual units" or just "Records the slope of a beam in MEI virtual units." ? If the latter, this should be changed in the other descriptions as well.

Reply:
`data.MEASUREMENTSIGNED` allows all sorts of units.

Reply:
Hm. Right. OK